packages/x-components/tests/e2e/cucumber/filters/sliced-filters.spec.ts

```diff
@@ -3,7 +3,7 @@ import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
 let totalFilters: number, hiddenFilters: number;
 
 Given('following config: max of sliced filters {int}', (slicedFiltersMax: number) => {
-  cy.visit('/?useMockedAdapter=true');
+  cy.visit('/?');
```

Comment:
I guess we can also remove the `?` here, right? Same for all the other files where you've applied this change (I'll just leave code suggestions in case you want to batch commit them instead of changing them manually):

```suggestion
  cy.visit('/');
```

Reply:
Global replaces in the morning do not work well